src/contribute-to-tidb/report-an-issue.md

```diff
@@ -0,0 +1,48 @@
+# Report an Issue
+
+If you think you have found an issue in TiDB, you can report it to the [issue tracker](https://github.com/pingcap/tidb/issues). If you would to like to report issues to TiDB documents or this development guide, they are in separate GitHub repositories, so you need to file issues to corresponding issue tracker, [TiDB document issue tracker](https://github.com/pingcap/docs/issues) and [TiDB Development Guide issue tracker](https://github.com/pingcap/tidb-dev-guide/issues). Read [Write Document](write-document.md) for more details.
+
+## Checking if an issue already exists
+
+The first step before filing an issue report is to see whether the problem has already been reported. You can [use the search bar to search existing issues](https://docs.github.com/en/github/administering-a-repository/finding-information-in-a-repository/using-search-to-filter-issues-and-pull-requests). This doesn't always work, and sometimes it's hard to know what to search for, so consider this extra credit. We won't mind if you accidentally file a duplicate report. Don't blame yourself if your issue is closed as duplicated. We highly recommend if you are not sure about anything of your issue report, you can turn to [internal.tidb.io](https://internals.tidb.io) for a wider audience and ask for discussion or help.
+
+## Filing an issue
+
+If the problem you’re reporting is not already in the issue tracker, you can [open a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) with your GitHub account. TiDB uses issue template for different kinds of issues. Issue templates are a bundle of questions to collect necessary information about the problem so it's easy for other contributors to participate. For example a bug report issue template consists of four questions:
+
+  - Minimal reproduce step.
+  - What did you expect to see?
+  - What did you see instead?
+  - What is your TiDB version?
+
+Answering these questions give the details about your problem so other contributors or TiDB users could pick up your issue more easily. 
+
+As previous section shows, duplicated issues should be reduced. To help others who encountered the problem find your issue, except for problem details answered in the issue template, a descriptive title which contains information that might be unique to it also helps. This can be the components related to or database feature used, the conditions that trigger the bug, or part of the error message if there is any. 
```

Comment:
I guess you meant `This can be the components related to database feature used` instead of `This can be the components related to or database feature used`?

Reply:
This sentence is from the current TiDB dev guide. I think it's meaning is "components related to your issue or database feature related to your issue". Let's rephrase the wording.

Reply:
Ok, I see.